packages/styleguide/src/components/Typography/Flyer.tsx

```diff
@@ -328,12 +328,9 @@ const A = forwardRef<HTMLAnchorElement, BaseProps>(
 
 export const Link: React.FC<any> = ({ children, href, ...props }) => {
   const { Link } = useRenderContext()
-  if (!href) {
-    return React.Children.only(children)
-  }
   return (
     <Link href={href} passHref>
-      <A href={href} {...props}>
+      <A href={href || '#restricted'} {...props}>
```

Comment:
Is `#restricted` still necessary?

```suggestion
      <A href={href} {...props}>
```

Reply:
Yeah… you are right. We don't need this. I just thought `#restricted` might make our DOM more clear, semantically.